src/MOI_wrapper.jl

```diff
@@ -28,6 +29,7 @@ function Solution()
         NaN,
         NaN,
         NaN,
+        0,
```

Comment:
This should probably be reset in `empty!`

Reply:
I think this https://github.com/jump-dev/ECOS.jl/blob/master/src/MOI_wrapper.jl#L135 takes care of that